lib/user_space/processor.c

```diff
@@ -1047,6 +1047,14 @@ static void processor_getcpu_fini(void)
 {
 }
 
+static void processor_cpuid_reg_get(unsigned opc, unsigned *r)
```

Comment:
I'd rather vote for this code and function to go away when it's not needed any more, than to try to add the implementations for different platforms. (As far as I understood, this is only a temporary solution with sleep anyway.)

From https://stackoverflow.com/a/51911779/3812944:
> It's probably not feasible to make a perfect hypervisor that is impossible to detect even with timing attacks, but over time it will likely also get harder and harder to detect a hypervisor which is meant to be stealthy (like blue pill).